Disable Travis Windows build?

Somethings on Travis Windows environment change lead to Clippy
cannot be built on it anymore. Here is the relevant part in [log](https://api.travis-ci.com/v3/job/272779719/log.txt):

```
  = note: C:\Users\travis\.rustup\toolchains\master\lib\rustlib\x86_64-pc-windows-gnu\lib\crt2.o:crtexe.c:(.rdata$.refptr.__onexitbegin[.refptr.__onexitbegin]+0x0): undefined reference to `__onexitbegin'
          C:\Users\travis\.rustup\toolchains\master\lib\rustlib\x86_64-pc-windows-gnu\lib\crt2.o:crtexe.c:(.rdata$.refptr.__onexitend[.refptr.__onexitend]+0x0): undefined reference to `__onexitend'
          collect2.exe: error: ld returned 1 exit status
```

So should we remove that Windows job or just disable it?

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,7 +26,13 @@ env:
 
 before_install:
   - export CARGO_TARGET_DIR="$TRAVIS_BUILD_DIR/target"
-  - curl -sSL https://sh.rustup.rs | sh -s -- -y --default-toolchain=nightly --profile=minimal
+  - |
+    case "$TRAVIS_OS_NAME" in
+      linux ) HOST=x86_64-unknown-linux-gnu;;
+      osx ) HOST=x86_64-apple-darwin;;
+      windows ) HOST=x86_64-pc-windows-msvc;;
+    esac
+  - curl -sSL https://sh.rustup.rs | sh -s -- -y --default-host="$HOST" --default-toolchain=nightly --profile=minimal
   - export PATH="$HOME/.cargo/bin:$PATH"
 install:
   - |

